x/harvest/types/params.go

```diff
@@ -267,24 +267,28 @@ func (bl BorrowLimit) Equal(blCompareTo BorrowLimit) bool {
 
 // MoneyMarket is a money market for an individual asset
 type MoneyMarket struct {
-	Denom             string            `json:"denom" yaml:"denom"`
-	BorrowLimit       BorrowLimit       `json:"borrow_limit" yaml:"borrow_limit"`
-	SpotMarketID      string            `json:"spot_market_id" yaml:"spot_market_id"`
-	ConversionFactor  sdk.Int           `json:"conversion_factor" yaml:"conversion_factor"`
-	InterestRateModel InterestRateModel `json:"interest_rate_model" yaml:"interest_rate_model"`
-	ReserveFactor     sdk.Dec           `json:"reserve_factor" yaml:"reserve_factor"`
+	Denom                  string            `json:"denom" yaml:"denom"`
+	BorrowLimit            BorrowLimit       `json:"borrow_limit" yaml:"borrow_limit"`
+	SpotMarketID           string            `json:"spot_market_id" yaml:"spot_market_id"`
+	ConversionFactor       sdk.Int           `json:"conversion_factor" yaml:"conversion_factor"`
+	InterestRateModel      InterestRateModel `json:"interest_rate_model" yaml:"interest_rate_model"`
+	ReserveFactor          sdk.Dec           `json:"reserve_factor" yaml:"reserve_factor"`
+	AuctionSize            sdk.Int           `json:"auction_size" yaml:"auction_size"`
```

Comment:
Make an issue to address `AuctionSize` in a follow-on PR

Reply:
Added here: https://github.com/Kava-Labs/kava/issues/741